File: clients/javascript/src/models/kvGetGroupIn.ts
// this file is @generated





export interface KvGetGroupIn {
    name: string;
}

export const KvGetGroupInSerializer = {
    _fromJsonObject(object: any): KvGetGroupIn {
        return {
            name: object['name'],
            };
    },

    _toJsonObject(self: KvGetGroupIn): any {
        return {
            'name': self.name,
            };
    }
}

File: clients/rust/src/models/kv_get_group_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[non_exhaustive]
#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct KvGetGroupIn {
    pub name: String,
}

impl KvGetGroupIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
